### PR TITLE
Update rspec

### PR DESCRIPTION
**Pull request by costin-zaharia-sonarsource**

### CODE REVIEW

File: analyzers/rspec/cs/S5753_c#.html
Comment (pavel-mikula-sonarsource) [inline analyzers/rspec/cs/S5753_c#.html:59]: This link is here with `see` as well as on the bottom in `see` section. I think we can remove one of them (this one)

File: analyzers/rspec/vbnet/S5753_vb.net.html
Comment (pavel-mikula-sonarsource) [inline analyzers/rspec/vbnet/S5753_vb.net.html:42]: VB.NET version is missing the `web.config` part from C#

**Review by pavel-mikula-sonarsource — CHANGES_REQUESTED** [review]
Minor tweaks suggested for the updated RSPEC

File: analyzers/rspec/vbnet/S5753_vb.net.html
Comment (costin-zaharia-sonarsource) [reply analyzers/rspec/vbnet/S5753_vb.net.html:42]: Yes, these were added as comments 😄. I've now updated the spec.

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM, please don't merge this before the feature branch is merged

**Pull request merged**
